.github/workflows/build_and_deploy_hold.yml

```diff
@@ -24,52 +24,19 @@ jobs:
     secrets:
       BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
       BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+  
+  call_extract_versions:
+    uses: ./.github/workflows/extract_versions.yml
 
-  call_format_branch_name:
-    uses: ./.github/workflows/format_branch_name.yml
-
-  extract_versions:
-    runs-on: ubuntu-latest
-    outputs:
-      minor_version: ${{ steps.vars.outputs.minor_version }}
-      major_version: ${{ steps.vars.outputs.major_version }}
-    steps:
-      - name: extract major and minor substrings
-        id: vars
-        run: |
-          MINOR_VERSION="$(echo "${{ github.ref_name }}" | cut -d '.' -f 1,2)"  
-          echo $MINOR_VERSION
-          echo ::set-output name=minor_version::${MINOR_VERSION}
-          MAJOR_VERSION="$(echo "${{ github.ref_name }}" | cut -d '.' -f 1)"
-          echo $MAJOR_VERSION
-          echo ::set-output name=major_version::${MAJOR_VERSION}
-
-  should_deploy_major_version:
-    runs-on: ubuntu-latest
-    outputs:
-      deploy_major_version: ${{ steps.vars.outputs.deploy_major_version }}
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: allow for major version deployment if the next minor version from current tag does not exist
-        id: vars
-        run: |
-          MINOR_VERSION=$(echo "${{ github.ref_name }}" | cut -d '.' -f 2) 
-          MAJOR_VERSION=$(echo "${{ github.ref_name }}" | cut -d '.' -f 1)
-          NEXT_MINOR_VERSION=$(( $MINOR_VERSION + 1 ))
-          OUTPUT=$(git tag --list "$MAJOR_VERSION.$NEXT_MINOR_VERSION.*")
-          if [ -z $OUTPUT ]
-          then
-            echo ::set-output name=deploy_major_version::true
-          fi
+  call_should_deploy_major_version:
+    uses: ./.github/workflows/should_deploy_major_version.yml
 
   call_deploy_tag_version:
     needs: 
       - call_unit_test
       - call_format_branch_name
```

Comment:
should remove this since we don't need it, or this would error out without the job defined

Reply:
ah, ya I can remove this